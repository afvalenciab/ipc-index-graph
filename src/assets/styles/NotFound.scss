// 04C0FC
.container--notfound {
  height: 100vh;
  background: #131D39;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;

  h1 {
    margin: 0;
    font-size: 100px;
    font-weight: 300;
    letter-spacing: 5px;
    color: #04C0FC;
    animation-name: beat;
    animation-duration: 1s;
    animation-iteration-count: infinite;
    animation-direction: alternate;
  }

  h2 {
    margin: 0;
    font-size: 40px;
    font-weight: 900;
    color: #FFB507;
    letter-spacing: 10px;
  }

  h3 {
    margin: 0;
    font-size: 20px;
    color: #fff;
    font-weight: 600;
    letter-spacing: 1.5px;
  } 
}

@keyframes beat {
  0% {
    opacity: 1;
    transform: scale(1);
  }
  100%{
    opacity: .2;
    transform: scale(.5);
  }
}