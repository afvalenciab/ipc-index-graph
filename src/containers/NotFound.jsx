import React from 'react';
import '../assets/styles/NotFound.scss';

const NotFound = () => {
  return (
    <main className='container--notfound'>
      <h1>404</h1>
      <h2>Error</h2>
      <h3>Page Not Found</h3>
    </main>
  );
};

export default NotFound;
