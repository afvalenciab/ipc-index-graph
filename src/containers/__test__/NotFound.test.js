import React from 'react';
import { create } from 'react-test-renderer';
import ProviderMock from '../../__mocks__/ProviderMocks';
import NotFound from '../NotFound';

describe('NotFound testing', () => {
  test('Match Snapshot', () => {
    const notFound = create(
      <ProviderMock>
        <NotFound />
      </ProviderMock>
    );
    expect(notFound.toJSON()).toMatchSnapshot();
  });
});
