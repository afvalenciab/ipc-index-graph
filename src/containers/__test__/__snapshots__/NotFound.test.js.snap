// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound testing Match Snapshot 1`] = `
<main
  className="container--notfound"
>
  <h1>
    404
  </h1>
  <h2>
    Error
  </h2>
  <h3>
    Page Not Found
  </h3>
</main>
`;
